### PR TITLE
Update python install step for python 3

### PR DESCRIPTION
**Pull request by gdebunne**
urllib and print() syntax updated

### What does this PR do?
Update the README for python 3

### What issues does this PR fix or reference?
None

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -156,7 +156,7 @@ or with python version 3:
 
 .. code:: console
 
-  python -c 'import urllib.request; print(urllib.request.urlopen("https://bootstrap.saltstack.com").read())' > bootstrap-salt.sh
+  python -c 'import urllib.request; print(urllib.request.urlopen("https://bootstrap.saltstack.com").read().decode("ascii"))' > bootstrap-salt.sh
```
Comment (vutny) [inline README.rst]: How about to change binary name to `python3`? Most distributions provide it like that.

Comment (gdebunne) [reply README.rst]: I'm ok with it, if it is more idiomatic.
python links to python3, which links to python3.7 on debian buster

File: README.rst [hunk 2]
```diff
@@ -152,6 +152,11 @@ All Python versions should support the following in-line code:
   python -c 'import urllib; print urllib.urlopen("https://bootstrap.saltstack.com").read()' > bootstrap-salt.sh
   sudo sh bootstrap-salt.sh git develop
 
+or with python version 3:
```
Comment (bryceml) [inline README.rst]: I would change the line up above that says "All Python Version should support" and make it more obvious/organized that one is python2 and one is python3.  I'd add the sudo line so they look more similar as well.

**Comment by s0undt3ch** [comment]
This has been merged on a separate PR

**Pull request closed without merging**